### ISSUE TITLE
Styling a specific language

### ISSUE THREAD

**Issue by TiagoPerdigao1983** [wiziple/gatsby-plugin-intl#171]
Since writing in German takes more space than in English, is it possible to use CSS to style only one specific language, like German for example?

My Gatsby config file: 

module.exports = {
  plugins: [
    {
      resolve: `gatsby-plugin-intl`,
      options: {
        // language JSON resource path
        path: `${__dirname}/src/intl`,
        // supported language
        languages: [`de`, `en`, `pt`],
        // language file path
        defaultLanguage: `de`,
      },
    },
  ],
};

I use a single style file for CSS.

Thank you for your help.

**Comment by dexterurbanski** [comment]
yes, you can use this https://css-tricks.com/almanac/selectors/l/lang/
